sample/WebApi.OutputCache.V2.Demo/WebApi.OutputCache.V2.Demo.csproj

```diff
@@ -9,7 +9,7 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>WebApi.OutputCache.V2.Demo</RootNamespace>
     <AssemblyName>WebApi.OutputCache.V2.Demo</AssemblyName>
-    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.6.2</TargetFrameworkVersion>
```

Comment:
thanks for the PR - but what is the purpose of this change? ASP.NET Web API can be used with .NET 4.5, so why bump this library?